spec/models/pacbio/library_factory_spec.rb

```diff
@@ -114,20 +113,28 @@
       end
 
       context 'when save errors' do
+        subject(:factory) { described_class.new(attributes) }
+
         before do
-          @factory = described_class.new(attributes)
-          allow(@factory).to receive(:valid?).and_return true
+          # This test seems to be designed to catch edge cases, in which problems that
```

Comment:
I'm not sure what this means. Not clear why we need to stub out the valid method. Is it just a spurious test that can be removed?

Reply:
Possibly... certainly as it stands its not hugely useful, but I wasn't really certain why it had been written. 

I think it is meant to be the belt to the braces of the validation (Ie. make sure nothing happend, even if we had failed to handle something explicitly), but I think there are better tests we could put in its stead, which is why I left the todo in there. Essentially it probably makes sense:

1) To check that nothing happens if either resource is invalid
2) To check that nothing happens if one of the saves fails for other reasons

I left it for now, as this was mostly something unrelated that rubocop picked up, and I wanted to keep this PR focused, hence popping in the issue alongside the todo.